packages/css/chip.css

```diff
@@ -20,15 +27,20 @@
   cursor: pointer;
   display: inline-flex;
   font-family: inherit;
+  gap: var(--gap);
+  line-height: var(--ds-line-height-sm);
+  margin: 0;
   min-height: var(--dsc-chip-height);
-  padding: var(--dsc-chip-padding);
+  padding: 0 var(--ds-spacing-3);
   text-decoration: none;
 
+  @composes ds-focus from './base/base.css';
+
   &:not([data-size]) {
     font-size: var(--ds-font-size-minus-1);
   }
 
-  /* Make focus ring also when input inside is focused */
+  /* Show focus ring when input inside is focused */
   &:has(:focus-visible) {
```

Comment:
Any reason we can't use `:focus-within`?
https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within

Reply:
Unfortunately no, because `focus-within` also shows up on mouse interaction; we want only on keyboard interaction ☺️

Reply:
Added comment about this in code ✅ 🤗